lib/sycamore/sycamore/transforms/detr_partitioner.py

```diff
@@ -640,19 +643,14 @@ def __init__(self, model_name_or_path, device=None):
     def _get_device(self) -> str:
         return choose_device(self.device, detr=True)
 
-    def infer(self, images: List[Image.Image], threshold: float) -> List[List[Element]]:
-        results = []
-        inputs = self.processor(images=images, return_tensors="pt").to(self._get_device())
-        outputs = self.model(**inputs)
-        target_sizes = torch.tensor([image.size[::-1] for image in images])
-        results.extend(
-            self.processor.post_process_object_detection(outputs, target_sizes=target_sizes, threshold=threshold)
-        )
+    def infer(self, images: List[Image.Image], threshold: float, use_cache: bool = False) -> List[List[Element]]:
+        if use_cache and self.cache:
+            print("Checking Cache for Image to JSON")
```

Comment:
May not want to keep this print and the one after.  There are already messages for both the hit and miss cases.

Reply:
updated